extensions/hibernate-reactive/deployment/src/main/java/io/quarkus/hibernate/reactive/deployment/HibernateReactiveProcessor.java

```diff
@@ -83,6 +85,8 @@ void registerBeans(BuildProducer<AdditionalBeanBuildItem> additionalBeans, Combi
             List<PersistenceUnitDescriptorBuildItem> descriptors,
             JpaModelBuildItem jpaModel) {
         if (!hasEntities(jpaModel)) {
+            LOGGER.warnf("Skipping registration of %s and %s because no JPA entities were found",
```

Comment:
This warning is a good idea IMO, but I believe the actual place where we "automatically disable" Hibernate Reactive is `io.quarkus.hibernate.reactive.deployment.HibernateReactiveProcessor#buildReactivePersistenceUnit`, so I'd move the warning there?

Also, Hibernate ORM has the exact same issue, I believe? Might be worth a warning in the Hibernate ORM extension too (in `io.quarkus.hibernate.orm.deployment.HibernateOrmProcessor#configurationDescriptorBuilding`).

Reply:
Thank you for your comments @yrodiere. The warn location was indeed not ideal so I pushed something new. Does the latest version look better?

I'll squash the commit when everyone's happy with the warnings 😃